style.css

```diff
@@ -0,0 +1,989 @@
+* {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  display: flex;
+  background-color: black;
+  justify-content: space-between;
+  z-index: 10;
+}
+
+nav ul,
+footer ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0 10px;
+}
+
+nav li,
+footer li {
+  color: rgb(163, 162, 162);
+  margin: 8px;
+  font-size: 13px;
+}
+
+.nav-2 li {
+  color: rgb(163, 162, 162);
+  margin: 15px 10px;
+  font-size: 14px;
+}
+
+.f-title li {
+  color: #f42;
+  padding: 0 10px;
+}
+
+.group-list li {
+  color: #f42;
+  font-size: 12px;
+  border-bottom: 1px solid #eee;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: #f42;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-x{
  color: #f42;
}
```
**Html file**

```
<div class="color-x">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_